src/org/traccar/sms/SMSException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 Anton Tananaev (anton@traccar.org)
+ * Copyright 2018 Andrey Kunitsyn (andrey@traccar.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.sms;
+
+
+import org.traccar.notification.NotificationException;
+
+public class SMSException extends NotificationException {
```

Comment:
Naming convention.

Reply:
Also, do we really need a separate SMS exception class? Can't we just use `NotificationException`?

Reply:
I'm not sure, we use SMS also for commands. Would it be good to combine it?

Reply:
Then it shouldn't inherit from `NotificationException`. Or maybe we should rename `NotificationException` to something more generic.

Reply:
I tied myself into knots. Could not make up better than `SendingException`. `CommunicationException` already used somewhere in ldap.
Removing inheritance will produce unneeded try/catch to convert one to another.

Reply:
Is there any specific reason why we need our own exceptions in the first place? Could we just use `RuntimeException` or some other standard `Exception` type?

Reply:
I think there is no reason... Runtime is to generic I think, what about `IOException` ?

Reply:
Maybe just create a single `MessageException` or `TransmissionException` or something like that?